content/My favorite extensions for programming Ruby in Visual Studio Code.md

```diff
@@ -0,0 +1,106 @@
+# Choosing a text editor
+
+It may be a tough decision, as a programmer, the text editor that you choose will be your main tool to do your work, and you will use it every day in your work-life.
+
+Everybody has their favorite text editor, in my life I have known people who use **Vim, Notepad++, Sublime Text, Atom**, and many others.
+
+I tried different text editors in my career, always looking for a perfect one -- at first I started with SublimeText, later the pretty UI of Atom hook me, and for the last year I have been coding with **Visual Studio Code**.
+
+I was enjoying Atom, but in some cases, my computer at the time couldn't handle it very well and it was getting a bit laggy to open the text editor or work with it. So a friend of mine told me that maybe Visual Studio Code could fit my needs, and he was right.
+
+# I use Visual Studio Code too! Tell me about the extensions
+
+## [GitLens](https://marketplace.visualstudio.com/items?itemName=eamodio.gitlens)
+
+This extension includes a lot of cool functionalities of Git in Visual Studio Code "GitLens simply helps you **better understand code**" says Eric Amodio, the creator of the extension.
+
+I love Gitlens because it permits me to detect who wrote the code that I am reading, this is really useful because when I have doubts about the code, I know who to talk to.
```

Comment:
```suggestion
I love Gitlens because it allows me to detect who wrote the code that I am reading, and this is really useful because when I have doubts about the code I know who to talk to.
```